features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -757,6 +757,10 @@
         Supported versions : IS 5.8.0 onwards
         -->
         <Introspection>
+            <!--
```

Comment:
If this is deprecated, can't we remove it?

Reply:
Removed in https://github.com/wso2/carbon-identity-framework/pull/3468/commits/8899908b08e0d5bbaf772ce2763f4e3ce87bc323